programs/programs/ma/dhsp_afterschool/calculator.py

```
@@ -0,0 +1,46 @@
+    Note: Tuition is on a sliding scale based on household size and income.
+    Some programs are free for eligible grades (e.g., free tuition for 6-8th graders
+    enrolled in CYP afterschool).
+    """
+
+    eligible_city = "Cambridge"
+    # K-8 typically covers ages 5-14 (Kindergarten through 8th grade)
+    min_child_age = 5
+    max_child_age = 14
+    # ~$900/month benefit value (reduced childcare cost)
+    amount = 900 * 12  # Annual value
+    dependencies = ["zipcode", "household_size"]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)